Cargo.lock

```diff
@@ -314,7 +314,7 @@ dependencies = [
 
 [[package]]
 name = "ipc-channel"
-version = "0.6.0"
+version = "0.5.1"
```

Comment:
There's a bug in ipc-channel 0.6 preventing us from landing the update in Servo, so we had to revert the update in WR (for now).